### PR TITLE
Add setting to customize notebook markdown preview line height

### PR DESCRIPTION
**Pull request by lucyzhang929**
This PR fixes https://github.com/microsoft/azuredatastudio/issues/8978.

Left: line height = 1
Right: original (ADS version 1.25.1)
Screen Shot 2021-01-06 at 11 33 03 AM

One thing to note:

1. Set line height to 1
2. Create new text cell
3. Type "hello"
4. Hit Enter
5. Type "world"
6. Exit cell

As shown in the GIF, space is added between "hello" and "world" upon exiting the cell. This is because they are converted to two separate p Elements and each element has margin space around it. In order to keep them as one p Element, you need to hit Shift+Enter instead of Enter.

hello_world
**Question** - do we want to customize margin space between p elements as well?

### CODE REVIEW

File: src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts [hunk 1]
```diff
@@ -121,6 +123,10 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this._register(this._configurationService.onDidChangeConfiguration(e => {
 			this.doubleClickEditEnabled = this._configurationService.getValue('notebook.enableDoubleClickEdit');
 		}));
+		this._register(this._configurationService.onDidChangeConfiguration(e => {
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts:126]: Just add this to the onDidChangeConfiguration above - no reason to have two separate callbacks.

**Review by Charles-Gagnon — APPROVED** [review]

**Review by corivera — APPROVED** [review]

**Review by VasuBhog — COMMENTED** [review]
Just a quick question, how does the user change the line-height exactly? Is it customizable to the user? 

I see from your gif they can use Shift+Enter for keeping as one p element.

**Comment by lucyzhang929** [comment]
@VasuBhog Yep! I added a setting to let the user change the line height. The default setting is 1.5 which is the same as the height before. 

Screen Shot 2021-01-06 at 6 17 47 PM

**Review by barbaravaldez — APPROVED** [review]

**Review by halerankin — APPROVED** [review]
I like this approach!

**Comment by VasuBhog** [comment]
> @VasuBhog Yep! I added a setting to let the user change the line height. The default setting is 1.5 which is the same as the height before.
> 
> Screen Shot 2021-01-06 at 6 17 47 PM

**Comment by VasuBhog** [comment]
Sorry misclick!

**Review by VasuBhog — APPROVED** [review]
Looks good!

**Review by MaddyDev — APPROVED** [review]

**Pull request merged**